assets/files.wxs

```diff
@@ -4,6 +4,57 @@
   <?define FileArchitecture = "$(env.FileArchitecture)" ?>
     <Fragment>
         <DirectoryRef Id="$(var.ProductDirectoryName)">
+            <Component Id="cmpCBE3857586454EB3B634B118E8EBD1D1" Guid="{CBE38575-8645-4EB3-B634-B118E8EBD1D1}">
+                <File Id="filCBE3857586454EB3B634B118E8EBD1D1" KeyPath="yes" Source="$(env.ProductSourcePath)\Microsoft.Win32.SystemEvents.dll" />
+            </Component>
```

Comment:
I lost understanding - why do we update the file manually? Can we generate it automatically?

Reply:
https://github.com/PowerShell/PowerShell/issues/6978

Reply:
The WiX tools will break patching.  When you build, I added errors to tell you what to add or remove, but didn't fully automate the update.  I updated the issue with the requirements of the update.